routes/products.test.ts

```diff
@@ -0,0 +1,15 @@
+import Server from '../source/server'
+import SuperTest from 'supertest'
+
+//close the server after test
+afterEach((done)=>{
+    Server.close();
+    done()
+})
+
+describe('routes/products', () =>{
+    it("pass", async () =>{
+        const res = await SuperTest(Server).get('/get');
+        expect(res.status).toEqual(200)
+    })
```

Comment:
Since you only assert the `status` of the response, then can you please change the description like,
`it('should return status 200')`

Reply:
sorry sir di pala ito kasali sa pag pull, tangalin ko na lang

Reply:
@ButchLendio Got it. Thank you